lte/gateway/docker/docker-compose.yaml

```diff
@@ -19,6 +19,8 @@ x-standard-volumes: &volumes_anchor
   - /tmp:/tmp
   - /var/log:/var/log
   - /etc/openvswitch:/etc/openvswitch
+  - /home/vagrant/magma/lte/gateway/configs:/magma/lte/gateway/configs
+  - /sbin/dhclient:/sbin/dhclient
```

Comment:
I think we should include `dhclient` in the services' docker images if it's needed, not mount it from the host.

Reply:
Done.